### PR TITLE
Create incident/update component status only after specified amount

### PR DESCRIPTION
**Pull request by jacekszubert**
of failed connection trials

### CODE REVIEW

File: cachet_url_monitor/configuration.py [hunk 1]
```diff
@@ -176,10 +179,27 @@ def __repr__(self):
...
 
+    def if_trigger_update(self):
+        """
+        Checks if update should be triggered - trigger it for all operational states
+        and only for non-operational ones above the configured thereshold (allowed_fails).
+        """
+
+        if self.status != 1:
+            self.current_fails = self.current_fails + 1
+            self.logger.info('Failure #%s with thershold set to %s' % (self.current_fails, self.allowed_fails))
```
Comment (mtakaki) [inline cachet_url_monitor/configuration.py]: You got a typo here (`thershold`), should be `threshold`.

**Comment by jacekszubert** [comment]
fixed :+1:

File: cachet_url_monitor/configuration.py [hunk 2]
```diff
@@ -176,10 +179,27 @@ def __repr__(self):
         del temporary_data['cachet']['token']
         return dump(temporary_data, default_flow_style=False)
 
+    def if_trigger_update(self):
+        """
+        Checks if update should be triggered - trigger it for all operational states
+        and only for non-operational ones above the configured thereshold (allowed_fails).
```
Comment (mtakaki) [inline cachet_url_monitor/configuration.py]: Ha, sorry, another typo here :)
You have `thereshold`.

Comment (jacekszubert) [inline cachet_url_monitor/configuration.py]: Haha, sorry. It's a really difficult word ;)

**Pull request merged**